<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>998</width>
    <height>778</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QComboBox" name="type_element">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>50</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="text_type">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>50</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Тип элемента</string>
   </property>
  </widget>
  <widget class="QLabel" name="number_element">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>50</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Количество элементов</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="input_table">
   <property name="geometry">
    <rect>
     <x>207</x>
     <y>180</y>
     <width>741</width>
     <height>231</height>
    </rect>
   </property>
   <row>
    <property name="text">
     <string>Сортамент</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>№ сечения</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>Сталь</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>Расчетная длина x (Lx), м</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>Расчетная длина y (Ly), м</string>
    </property>
   </row>
   <column>
    <property name="text">
     <string>1</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>2</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>3</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>4</string>
    </property>
   </column>
   <item row="2" column="0">
    <property name="text">
     <string/>
    </property>
   </item>
  </widget>
  <widget class="QTableWidget" name="output_table">
   <property name="geometry">
    <rect>
     <x>207</x>
     <y>417</y>
     <width>741</width>
     <height>271</height>
    </rect>
   </property>
   <row>
    <property name="text">
     <string>N(+), кг</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>N(-), кг</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>Ry, кг/см2</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>A, см2</string>
    </property>
   </row>
   <column>
    <property name="text">
     <string>1</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>2</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>3</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>4</string>
    </property>
   </column>
  </widget>
  <widget class="QListWidget" name="listWidget">
   <property name="geometry">
    <rect>
     <x>6</x>
     <y>50</y>
     <width>194</width>
     <height>634</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="solve_button">
   <property name="geometry">
    <rect>
     <x>567</x>
     <y>736</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Расчет</string>
   </property>
  </widget>
  <widget class="QPushButton" name="word_button">
   <property name="geometry">
    <rect>
     <x>657</x>
     <y>736</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>В Word</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_3">
   <property name="geometry">
    <rect>
     <x>747</x>
     <y>736</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Сохранить</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="text_folder">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>10</y>
     <width>621</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>`</string>
   </property>
   <property name="cursorPosition">
    <number>1</number>
   </property>
   <property name="dragEnabled">
    <bool>false</bool>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="button_folder">
   <property name="geometry">
    <rect>
     <x>840</x>
     <y>10</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>...</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="number">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>50</y>
     <width>51</width>
     <height>31</height>
    </rect>
   </property>
   <property name="minimum">
    <number>1</number>
   </property>
   <property name="maximum">
    <number>8</number>
   </property>
  </widget>
  <widget class="QLabel" name="picture">
   <property name="geometry">
    <rect>
     <x>790</x>
     <y>50</y>
     <width>151</width>
     <height>101</height>
    </rect>
   </property>
   <property name="text">
    <string>Picture</string>
   </property>
  </widget>
  <widget class="QComboBox" name="type_section">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>100</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="text_section">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>100</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Тип сечения</string>
   </property>
  </widget>
  <widget class="QLabel" name="text_code">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>100</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Нормы</string>
   </property>
  </widget>
  <widget class="QComboBox" name="type_code">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>100</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="text_error">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>678</x>
     <y>142</y>
     <width>268</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>`</string>
   </property>
   <property name="cursorPosition">
    <number>1</number>
   </property>
   <property name="dragEnabled">
    <bool>false</bool>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="9"/>
 <resources/>
 <connections/>
 <designerdata>
  <property name="gridDeltaX">
   <number>10</number>
  </property>
  <property name="gridDeltaY">
   <number>10</number>
  </property>
  <property name="gridSnapX">
   <bool>false</bool>
  </property>
  <property name="gridSnapY">
   <bool>false</bool>
  </property>
  <property name="gridVisible">
   <bool>true</bool>
  </property>
 </designerdata>
</ui>
